handlers/images.go

```diff
@@ -151,3 +154,46 @@ func ImageManifest(w http.ResponseWriter, r *http.Request) {
 
 	respondWithJSON(w, http.StatusOK, image.Seed)
 }
+
+func JITImageManifest(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	regUrl := vars["registry"]
+	imgstr := vars["image"]
+	org := ""
+	if strings.Contains(regUrl, "docker.io") || regUrl == "hub.docker.com" {
+		regUrl = "hub.docker.com"
+		temp := strings.SplitN(imgstr, "/", 2)
+		org = temp[0]
+		imgstr = temp[1]
+	}
+	reg, err := registry.CreateRegistry(regUrl, org, "", "")
```

Comment:
Since we don't provide any way to pass credentials this will not work against any registry requiring authentication. Can we create a subsequent issue to add that support?

Reply:
Done